tuxedo/charts/tuxedo-ws/values.yaml

```
@@ -0,0 +1,83 @@
+# Copyright (c) 2022, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# Default values for tuxedo-ws.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: iad.ocir.io/paasdevblockchain/oracle/tuxedows_svr
```

```suggestion
  repository: 
```

Thankfully, this URL doesn't actually work because that would have been a much bigger issue.